docs/versioning_integration.rst

```diff
@@ -312,6 +314,69 @@ in the form of a dictionary of {model_name: {field: method}}, the admin for the
 Given the code sample above, "This is how we add" would be displayed as
 "this is how we add extra field text!" in the changelist of PostAdmin.
 
+Adding status indicators to a versioned content model
+-----------------------------------------------------
+
+djangocms-versioning provides status indicators for django CMS' page content models as you know them from the page tree:
+
+.. image:: static/Status-indicators.png
+    :width: 50%
+
+You can use these on your content model's changelist view admin by adding the following fixin to the model's Admin class:
+
+.. code-block:: python
+
+    import json
+    from djangocms_versioning.admin import StateIndicatorAdminMixin
+
+
+    class MyContentModelAdmin(StateIndicatorAdminMixin, admin.Admin):
+        # Adds "indicator" to the list_items
+         list_items = [..., "state_indicator", ...]
+
+.. note::
+
+    For grouper models the mixin expects that the admin instances has properties defined for each extra grouping field, e.g., ``self.language`` if language is an extra grouping field.
+
+    This is typically set in the ``get_changelist_instance`` method, e.g., by getting the language from the request. The page tree keeps its extra grouping field (language) as a get parameter.
```

Comment:
Should this still refer to page tree or Content Model?

Reply:
Yes, it is actually an example on how to keep track of additional grouping fields.